playground/src/screens/OptionsScreen.tsx

```diff
@@ -66,6 +67,7 @@ export default class Options extends React.Component<Props> {
           onPress={this.pushButtonsScreen}
         />
         <Button label="StatusBar" onPress={this.statusBarScreen} />
+        <Button testID={GOTO_SEARCHBAR_SCREEN} label="Search Bar" onPress={this.searchBarScreen} />
```

Comment:
You can add `platform={'ios'}` so this button is visible only on iOS. There's no need to even show it on Android.